src/emit/instructions.rs

```diff
@@ -867,69 +869,13 @@ pub(super) fn process_instruction<'a, T: TargetRuntime<'a> + ?Sized>(
                 (ptr, len)
             };
 
-            let seeds = if let Some(seeds) = seeds {
-                let len = seeds.ty().array_length().unwrap().to_u64().unwrap();
-                let seeds_ty = bin.llvm_type(
-                    &Type::Slice(Box::new(Type::Slice(Box::new(Type::Bytes(1))))),
-                    ns,
-                );
+            let seeds_ty =
+                Type::Slice(Type::Slice(Type::Slice(Type::Bytes(1).into()).into()).into());
```

Comment:
Why did the type go from slice of slice (`Type::Slice(Type::Slice(..))`) to slice of slice of slice (`Type::Slice(Type::Slice(Type::Slice(..)))`)?

Reply:
Doesn't `invoke_signed_c` require a slice of slice?

Reply:
A single seed is a slice of bytes.
There are multiple seeds allowed for an address (essentially hash of (program_id + seeds1 + seed2)
A single call can sign for multiple addresses.
That's three slices.

I'll add a comment.